geometry_msgs/Pose      Camera_Pose 
geometry_msgs/PoseArray Robot_Pose
geometry_msgs/PoseArray Obstacle_Pose
